README.md

```diff
@@ -14,7 +14,7 @@ For example:
 If you are building with Gradle, simply add the following line to the dependencies section of your build.gradle file:
 
 ```
-implementation 'com.discord:simpleast:1.0.0'
+implementation 'com.discord:simpleast:1.1.1'
```

Comment:
figured I'd bump this bad boy too, but I can rip out into a different commit as well.